File: quatorze/quatorze.py
word = input("Digite uma palavra: ")
vowels_count = 0

for letter in word:
    if letter.lower() in "aeiou":
        vowels_count += 1

replace_letter = input("Digite um caractere para substituir as vogais: ")
new_word = word.replace("a", replace_letter).replace("e", replace_letter).replace("i", replace_letter).replace("o", replace_letter).replace("u", replace_letter)

print(f"A palavra '{word}' tem {vowels_count} vogais.")
print(f"A palavra '{new_word}' com as vogais substituídas por '{replace_letter}' é: {new_word}")

File: quatorze/quatorze.js
let word = prompt("Digite uma palavra:");
let vowels_count = 0;

for (let letter of word) {
  if ("aeiou".includes(letter.toLowerCase())) {
    vowels_count += 1;
  }
}

let replace_letter = prompt("Digite um caractere para substituir as vogais:");
let new_word = word.replaceAll(/[aeiou]/gi, replace_letter);

console.log(`A palavra '${word}' tem ${vowels_count} vogais.`);
console.log(`A palavra '${new_word}' com as vogais substituídas por '${replace_letter}' é: ${new_word}`);